alpaca/common/websocket.py

```diff
@@ -218,6 +229,13 @@ async def _dispatch(self, msg: Dict) -> None:
             )
             if handler:
                 await handler(self._cast(msg_type, msg))
+        elif msg_type == "n":
+            symbol = ",".join(symbol)
```

Comment:
If `msg.get("symbols", [])` always returns list, how about handle a symbol one by one? I do not think a use case of multi symbols handler (e.g. elf._handlers["news"].get("AAPL,META"))

Reply:
The only reason I added this line was because of the [documentation](https://docs.alpaca.markets/docs/streaming-real-time-news) `symbols	string[]	List of related or mentioned symbols`.
Your suggestion sounds practical, though I think we should utilize the flexibility of the api.
Do you mean to suggest the following:
```suggestion
            symbol = ",".join(symbol)[0]

```

Reply:
A user subscribes with specific handler (handerA) for AAPL and general handler (handerG) for IBM.
Then, a news has symbols == ['AAPL', 'IBM', 'TSLA']. A user may expect handerA and handerG are called.
Therefore, what I thought was below. we may need to avoid duplicated call for the handler '*' though.
```Python
for s in symbol:
    if symbol in self._handlers['news']:
        self._handlers['news'][symbol]
    else:
        self._handlers['news']['*']
```

Reply:
Please take a look to see if this is what your looking for.